diff --git "a/X:\\codex-studio\\docs\\codemap\\codemap.lock" "b/X:\\codex-studio\\docs\\codemap\\.staging\\codemap.lock"
index 1968749c..05533567 100644
--- "a/X:\\codex-studio\\docs\\codemap\\codemap.lock"
+++ "b/X:\\codex-studio\\docs\\codemap\\.staging\\codemap.lock"
@@ -2,8 +2,8 @@
   "schema_version": 2,
   "source_commit": "be3541666a65e277c584e3609acd65f81426fc61",
   "working_tree_dirty": true,
-  "generated_at": "2026-09-22T05:54:12Z",
-  "generation_id": "b8affa3fd5ab95e7cb68851c7ae6420db4d7246c91bc891fc80bc6ae568d9890",
+  "generated_at": "2026-09-22T06:13:56Z",
+  "generation_id": "64967d69123c2158b14f1c3281dbf818071da7ef7ed3a9fe05a8167099348d82",
   "analysis": {
     "version": "syntax-1",
     "digest": "277e322054c030f72811f774ad4e959bcf8d1244bfb70c076e298cf824ccdc3c"
@@ -2209,8 +2209,8 @@
         "path": "components/recipes/RecipeResultPreview.test.tsx",
         "origin": "tracked",
         "state": "present",
-        "digest": "bbfd88558860e2374d7380a351f2b3cae404e6af0e4ade26b68221e7b8b4ee5d",
-        "size": 6650
+        "digest": "a4997ea59fadb4d2e17734557a09644b3fbcc04412eb05b13169f71784baca0b",
+        "size": 7891
       },
       {
         "path": "components/recipes/RecipeResultPreview.tsx",
@@ -4281,8 +4281,8 @@
         "path": "hooks/useHorizontalDragScroll.ts",
         "origin": "untracked",
         "state": "present",
-        "digest": "079fa1692df9f7add1eaa224ec2afc583e6ee04ad7cae17c577a8dee8e3683b2",
-        "size": 2320
+        "digest": "552620f671cea25cb4398c6173b7b9667046625c1fb63c92e2b800c0e197d17e",
+        "size": 2032
       },
       {
         "path": "hooks/useImageInputSurface.ts",
@@ -7652,12 +7652,12 @@
         "size": 710
       }
     ],
-    "digest": "9812fe3f43c2ff4cafa7710465d6cf48a7a94a6c19bfaa36ccac2d57332906c1"
+    "digest": "ffce3eee168c9485833654faa19b64204112e400a603540d198769ace232b747"
   },
-  "source_digest": "f0bdc7d4b4a8cdc726b24519cd251c25cd897c1a282725f50dab21dabd7489cc",
+  "source_digest": "ba2f1f4085a9219ed6722cd70af6751ff5c3765dfd488c0f9bda14eb24e8a34f",
   "artifact_hashes": {
-    "codemap.json": "35daa92a1030b8de820466073140be076e354b7db87fd69826c20b2382d55dd6",
-    "codemap.md": "5263af124af68110eded45d5448d12e19c7a98f04bebaa5da871150a78a28a2f",
-    "codemap.html": "1c8ea91e1dee0e55af90df9897f8693037d6542c7cd13d3f6e6f515331f86254"
+    "codemap.json": "05638b16bf9e456362d8a8b7aba4260a1adf53a25040846bde01ae3e87a3f8bb",
+    "codemap.md": "dd7bfb1d43acf62b308482000ea1e91a7fc19e82f293194fc7c2ac840ba730bf",
+    "codemap.html": "f23fd715cf373987b6d5e8299db7745203406caa99b7b9f2091462199dcce359"
   }
 }
